Cesium.Parser.Tests/PreprocessorTests/PreprocessorTests.cs

```diff
@@ -33,4 +33,17 @@ int test()
 int test()
 {
 }", new() { ["foo.h"] = "void foo() {}", ["bar.h"] = "int bar = 0;" });
+
+
+    [Fact]
+    public async Task ErrorMsg()
+    {
+        NotSupportedException err = await Assert.ThrowsAsync<NotSupportedException>(async () => await DoTest(@"
```

Comment:
I think that it should be not a `NotSupportedException`, but something else. Let's create a new exception class for this? `class PreprocessorException : Exception`.

Reply:
Done!